src/locales/en.json

```diff
@@ -97,8 +97,9 @@
     "duration_header": "Duration",
     "duration_question": "How long is the activity, in minutes? (For a repeated activity: minutes per week?)",
     "intimate_risk_warning": "We have applied a minimum Activity Risk for kissing due to the risk involved in exchanging fluids.",
-    "person_risk_profile_question": "What is their risk profile?",
-    "person_risk_profile_header": "Risk Profile",
+    "behavior_question": "How did they spend their last 10 days?",
+    "behavior_header": "Behavior",
+    "person_risk_note": "Each nearby person has a person risk of <2>{{ personRisk }} $t(calculator.pointsdisplay.microCOVIDs)</2> this week. Person risk (chance they currently have COVID) is calculated from their location, behavior, vaccination status, and infection status.",
```

Comment:
Also, why did you say "this week" vs "currently"? ... When I read "this week" I think "oh, I don' need to go back and re-check this score for a week." When in fact, the prevalence could change tomorrow and their score would change.  I'm sure you were thinking of some edge case to help users understand something, I just don't see it yet. :)

Reply:
Upon further review "this week" also seems at odds with the prompt, which is asking about the last 10 days.